doc/source/examples/multiphysics/warming-up-a-viscous-fluid/warming-up-a-viscous-fluid.rst

```diff
@@ -322,6 +322,64 @@ After the fluid has been heated up by the right plate, the temperature is really
     :alt: Rescaled domain with temperature (t = 7)
     :width: 30%
 
+
+Horizontal domain
+~~~~~~~~~~~~~~~~~
+
+Several adjustments have to be made in the `.prm` to turn the domain clockwise, so that it becomes horizontal, with the upper wall being the no slip and thermal insulation boundary condition, and the lower wall with the flow in the y-direction (:math:`v=2`) and heating at Tw:
+
+* in ``subsection mesh``: ``set grid arguments = 0, 0 : 1, 0.5 : true``
+* in ``subsection analytical solution``, ``subsection temperature``: 
+   ``set Function expression = Tw+(((rho*nu)*v*v)/(2*K))*(1-(y/B)*(y/B))``
+* and most importantly, the ``boundary conditions`` subsection should state explicitly ``outlet`` for the fluid boundary conditions, as the ``number`` of boundary conditions should be adapted to use the bottom and top wall (see the `deal.II documentation on hyper_rectangle grid generator <https://www.dealii.org/current/doxygen/deal.II/namespaceGridGenerator.html#a56019d263ae45708302d5d7599f0d458>`_ for further details):
+
+.. code-block:: text
+
+	subsection boundary conditions
+	  set number                  = 4
+	    subsection bc 0
+	    set id = 0
+		set type              = outlet
+	    end
+	    subsection bc 1
+	    set id = 1
+		set type              = outlet
+	    end
+	    subsection bc 2
+	    set id = 2
+		set type              = noslip
+	    end
+	    subsection bc 3
+	    set id = 3
+		set type              = function
+		subsection u
+		    set Function expression = 2
+		end
+		subsection v
+		    set Function expression = 0
+		end
+	    end
+	end
+
+	subsection boundary conditions heat transfer
+	  set number                  = 4
+	    subsection bc 2
+	    set id = 2
+		set type          = convection-radiation
+		set h             = 0
+		set Tinf	  = 0
+	    end
+	    subsection bc 3
+	    set id = 3
+		set type              = temperature
+		set value             = 80
+	    end
+	end
+
+.. tip::
+	Heat transfer boundary conditions are of ``type = none`` by default, so this is the type implicitly used for ``subsection bc 0`` and ``subsection bc 1``.
```

Comment:
It's would be better to have the type=none be replaced by something like zero flux or 'natural' because that's what is being done when you don't do anything. I like no flux or zero flux, but I am open to alternatives.

Reply:
I have a preference for `noflux`, I will change that